### DISCUSSION TITLE
Help!! transfert copy OpenMage in new server but have error

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#1989] [Questions & Answers]
Hi everyone
I have a problem

i am transferring a copy of an openMage project from one server to another

The servers are identical

the problem is that on the new server the copy of OpenMage returns this error:

```
a:4:{i:0;s:40:"WRONGPASS invalid username-password pair";i:1;s:1123:"#0 /var/www/html/magento/lib/Credis/Client.php(825): Credis_Client->read_reply()
#1 /var/www/html/magento/lib/Credis/Client.php(588): Credis_Client->__call()
#2 /var/www/html/magento/lib/Cm/Cache/Backend/Redis.php(128): Credis_Client->auth()
#3 /var/www/html/magento/lib/Zend/Cache.php(153): Cm_Cache_Backend_Redis->__construct()
#4 /var/www/html/magento/lib/Zend/Cache.php(94): Zend_Cache::_makeBackend()
#5 /var/www/html/magento/app/code/core/Mage/Core/Model/Cache.php(142): Zend_Cache::factory()
#6 /var/www/html/magento/app/code/local/Mage/Core/Model/Config.php(1374): Mage_Core_Model_Cache->__construct()
#7 /var/www/html/magento/app/Mage.php(532): Mage_Core_Model_Config->getModelInstance()
#8 /var/www/html/magento/app/code/core/Mage/Core/Model/App.php(428): Mage::getModel()
#9 /var/www/html/magento/app/code/core/Mage/Core/Model/App.php(322): Mage_Core_Model_App->_initCache()
#10 /var/www/html/magento/app/code/core/Mage/Core/Model/App.php(364): Mage_Core_Model_App->baseInit()
#11 /var/www/html/magento/app/Mage.php(752): Mage_Core_Model_App->run()
#12 /var/www/html/magento/index.php(88): Mage::run()
#13 {main}";s:3:"url";s:1:"/";s:11:"script_name";s:10:"/index.php";}
```

i disabled redis, i cleared the cache, i disabled added plugins, but nothing

Password and User Name of db, redis, users is same of original
please help

**Comment by addison74** [comment]
Although at first glance you think the servers are identical, the error proves that you did not reproduce it properly on the second server. You should look for a tutorial on the Internet to explain how to transfer a Magento store to another server. As long as the database has access to the same user that is set in Magento, and Redis has the same configurations, it can't work. A dump from the terminal and import, plus copying files, nothing simpler. Then pay attention to permissions on the file system. Connect to Redis from Terminal and see what is stored there. When you turn off Redis mode in Magento (true-> false) do the sessions and cache appear in the var/cache and var/session directories?

**Reply by Danieleeffe1** [reply]
Thanks for the reply

i did a clean install of openmage and lesty_fpc plugins with the same login details and had no problems

**Reply by luigifab** [reply]
The message is explicit: `WRONGPASS invalid username-password pair` from `lib/Credis/Client`.
You have in `app/etc/local.xml` or in another XML configuration file something that use Redis, but this Redis configuration is wrong.
No?

**Reply by Sdfendor** [reply]
Did you disable redis on server side or in OpenMage? The error suggests the redis can't connect with the credentails your config provides (and you didn't change these). But if deactivated the lib wouldn't be called.

**Comment by Danieleeffe1** [comment]
I checked the lesty_fpc and Local.xml files several times in the end I had to reinstall clean OpenMage and insert one at a time all the plug-ins that were present in the backup and now everything works.

My suspicion is that some files were lost at the time of the backup.

the previous OpenMage installation was a Magento 1.9.4.5 with uploaded the openmage files to take advantage of the backup options.

I also performed the same operations in the new installation.

**Reply by Sdfendor** [reply]
Good to hear that you could resolve your issue!
